cmd/influx/pkg.go

```diff
@@ -23,16 +23,18 @@ import (
 	"gopkg.in/yaml.v3"
 )
 
-func pkgCmd() *cobra.Command {
-	cmd := pkgApplyCmd()
+type pkgSVCFn func(token, addr string, isLocal, skipVerify bool) (pkger.SVC, error)
```

Comment:
DI ftw

Reply:
I think we would like to group `token, addr string, isLocal, skipVerify bool` together as a single dependency, so when we would like to add more parameters, we only need to change one place. Also we don't have to write similar code for every one  else.

```go
     type CliRequest struct{
           Token string
            Addr  string
            IsLocal bool
            SkipVerify bool
     }

     func (c CliRequest) PkgService() *pkger.Service, error{ ... }
```

Also i think it's better if we put CliRequest in a separate  folder under cmd/influx/, other than the main package. The main package is oversized now